### ISSUE TITLE
Warcraft Recorder crashes with no explanation

### ISSUE THREAD

**Issue by aza547** [aza547/wow-recorder#264]
![image](https://user-images.githubusercontent.com/32389325/210133424-b33bdc6a-12e5-481b-baf2-e0a06d06f29c.png)
![image](https://user-images.githubusercontent.com/32389325/210133450-7558d1a9-d29d-4a19-ae09-a38cae1a30f8.png)
![image](https://user-images.githubusercontent.com/32389325/210133453-5b9882b3-0342-479f-8e76-f879f4adc288.png)
![image](https://user-images.githubusercontent.com/32389325/210133461-4ed14c29-5ed9-4603-b7eb-60edb9d99b0d.png)
![image](https://user-images.githubusercontent.com/32389325/210133471-a23ff654-44cd-4405-a165-08951b92cbf3.png)

Nothing of interest in logs.
[WarcraftRecorder-2022-12-30 (1).log](https://github.com/aza547/wow-recorder/files/10327177/WarcraftRecorder-2022-12-30.1.log)

**Comment by aza547** [comment]
Suspect this is caused by access rights on Windows 11 differing for those on Windows 10.

This seem to happen at a stage of disk interaction (strange that it's reading back files rather than writing them though?). 

Workaround is to run as admin. 

Might want to add something like the below [as per](https://stackoverflow.com/questions/37322862/check-if-electron-app-is-launched-with-admin-privileges-on-windows). 

```
"build": {
    "win": {
      "requestedExecutionLevel": "highestAvailable"
    }
```

**Comment by aza547** [comment]
Passed an alpha version of this build along to the user who reported it and it seems fixed. Will be released in 3.3.0.
